packages/eslint-plugin/lib/rules/no-nested-await.js

```diff
@@ -90,17 +90,15 @@ module.exports = {
         if (!isAwaitAllowedInNode(parent)) {
           context.report({
             node,
-            message:
-              'Unexpected `await` expression (not top of async function or `for-await-of` body)',
+            messageId: 'unexpectedNestedAwait',
           });
         }
       },
-      ForOfStatement: node => {
-        if (node.await === true && !isAwaitAllowedInNode(node.parent)) {
+      'ForOfStatement[await=true]': node => {
```

Comment:
neat feature of the rule runner

Reply:
Indeed; and even more at https://github.com/estools/esquery#readme .